test/cppsuite/tests/run.cxx

```diff
@@ -67,26 +67,20 @@ value_missing_error(const std::string &str)
 
 /*
  * Run a specific test.
- * config_name is the configuration name. The default configuration is used if it is left empty.
```

Comment:
Comment formatting is weird here.

Reply:
I am describing the role of each arg. One arg per line.